app.py

```diff
@@ -76,5 +147,15 @@ def is_currently_playing(handler_input: HandlerInput) -> bool:
     session_attr = handler_input.attributes_manager.session_attributes
     return session_attr.get('gameStarted') == True
 
+def ask_question() -> str:
```

Comment:
This needs to take `session_attr` as a parameter.

Also the docstring is not particularly helpful... maybe say something like: "Constructs an arithmetic equation given session attributes"